.github/workflows/deployment_uat.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
```

Comment:
trigger is iedere push & pull_request ?

Reply:
Hoe bepaal je de eindbestemming ? merge met uat -> uat omgeving ? merge met main -> prd omgeving ?